installing/install_config/customizations.adoc

```diff
@@ -122,19 +246,19 @@ that are based on many of these CRDs to add more functionality to your
 |AuthenticationOperatorConfig
 |authentication.operator.openshift.io
```

Comment:
looks like it has leaked here too.  @bergerhoffer please remove references to 

1. kubeapiserver.operator.openshift.io
2. kubecontrollermanager.operator.openshift.io
3. kubescheduler.operator.openshift.io
4. openshiftapiserver.operator.openshift.io
5. authentication.operator.openshift.io
6. serviceca.operator.openshift.io

These are not resources for cluster-admins.  This is not an area where they are allowed to alter configuration and status is reported back through our clusteroperator, so they do not need to view this resource for status either.

Reply:
this is commented out and doesn't appear in the rendered doc.  not sure why it was put here in the first place but i left it as a reference of the "global set of stuff".